client/src/mac.rs

```diff
@@ -0,0 +1,43 @@
+use serde::{Deserialize, Serialize};
+#[cfg(feature = "native_crypto")]
+use subtle::ConstantTimeEq;
+use zbus::zvariant::Type;
+
+#[derive(Deserialize, Serialize, Type, Debug, Clone)]
+pub struct Mac(Vec<u8>);
```

Comment:
Can you add a single line comment about this?

Reply:
Sure, however it requires a dummy `{}`